elastic/kibana
# [Synthetics] prevent origin from being included on legacy Synthetics integration policies

The key `origin` is used by Heartbeat to determine whether the monitor is sourced from the Uptime UI or a project. This key should not be present on legacy Synthetics Integration policies, but should be included on auto-generated policies created when using private locations.

### Testing
1. Navigate to Uptime and create a private location. No need to assign an agent to it
2. Create a monitor with this private location.
3. Navigate to the Elastic Synthetics Integration page. Click Add integration and create a second monitor tied to the same agent policy
4. Navigate to that agent policy and view the policy config
5. Ensure that the first monitor contains the `origin` key, while the second monitor does not contain the `origin` key.
